### PR TITLE
Switch long bigint

### PR DESCRIPTION
**Pull request by Zetazzz**
add a flag to switch Long or bigint
handle all cases switching between Long or bigint
also to switch helper.ts between Long or bigint

### CODE REVIEW

File: __fixtures__/v-next/outputv4/akash/deployment/v1beta1/deployment.ts [hunk 1]
```diff
@@ -135,42 +135,42 @@ export interface MsgCloseDeploymentResponseSDKType {}
...
   owner: string;
-  dseq: Long;
+  dseq: bigint;
 }
 
 /** DeploymentID stores owner and sequence number */
 export interface DeploymentIDSDKType {
   owner: string;
-  dseq: Long;
+  dseq: bigint;
```
Comment (pyramation) [inline __fixtures__/v-next/outputv4/akash/deployment/v1beta1/deployment.ts:144]: should this be `BigInt` with caps included?

File: __fixtures__/v-next/outputv4/akash/deployment/v1beta2/group.ts [hunk 2]
```diff
@@ -177,7 +177,7 @@ export const Group = {
     message.groupId = object.groupId !== undefined && object.groupId !== null ? GroupID.fromPartial(object.groupId) : undefined;
     message.state = object.state ?? 0;
     message.groupSpec = object.groupSpec !== undefined && object.groupSpec !== null ? GroupSpec.fromPartial(object.groupSpec) : undefined;
-    message.createdAt = object.createdAt !== undefined && object.createdAt !== null ? Long.fromValue(object.createdAt) : Long.ZERO;
+    message.createdAt = object.createdAt !== undefined && object.createdAt !== null ? (prop => BigInt(prop.toString!!()))(object.createdAt) : BigInt("0");
```
Comment (pyramation) [inline __fixtures__/v-next/outputv4/akash/deployment/v1beta2/group.ts:180]: what is the idea behind using an executed anonymous function?

File: __fixtures__/v-next/outputv4/akash/deployment/v1beta1/deployment.ts (hunk 1 above)
Comment (Zetazzz) [reply __fixtures__/v-next/outputv4/akash/deployment/v1beta1/deployment.ts:144]: Hi, I think bigint here is the way to use it as a basic type. For BigInt, it's more like a constructor for bigint. So for me I prefer bigint. 

And also, In fact at the beginning I used BigInt here, but my local compiler told me:

The left-hand side of an arithmetic operation must be of type 'any', 'number', 'bigint' or an enum type.  ts(2362)
Operator '*' cannot be applied to types 'BigInt' and 'bigint'.  ts(2365)

So I changed it into bigint.

File: __fixtures__/v-next/outputv4/akash/deployment/v1beta2/group.ts (hunk 2 above)
Comment (Zetazzz) [reply __fixtures__/v-next/outputv4/akash/deployment/v1beta2/group.ts:180]: Ah... I just thought it as a bigint version of Long.fromValue (without applying toString to the property itself). After thinking this again, i'll change it to a decent way of doing this.

Comment (Zetazzz) [reply __fixtures__/v-next/outputv4/akash/deployment/v1beta2/group.ts]: Done removing.

**Pull request merged**